components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.soap/src/main/java/org/wso2/carbon/event/input/adapter/soap/SOAPEventAdapterFactory.java

```diff
@@ -20,12 +20,22 @@
 
 import org.wso2.carbon.event.input.adapter.core.*;
 import org.wso2.carbon.event.input.adapter.soap.internal.util.SOAPEventAdapterConstants;
+import org.wso2.carbon.utils.CarbonUtils;
 
 import java.util.*;
 
 
```

Comment:
@lafernando 
Thank you for pointing out. Fixed in PR: https://github.com/wso2/carbon-analytics-common/pull/146